src/patternfly/components/ProgressStepper/progress-stepper.scss

```diff
@@ -75,11 +75,28 @@
   --pf-c-progress-stepper__step--m-current__step-title--FontWeight: var(--pf-global--FontWeight--bold);
   --pf-c-progress-stepper__step--m-current__step-title--Color: var(--pf-global--Color--100);
   --pf-c-progress-stepper__step--m-pending__step-title--Color: var(--pf-global--Color--200);
+  --pf-c-progress-stepper__step--m-danger__step-title--Color: var(--pf-global--danger-color--100);
   --pf-c-progress-stepper__step--m-danger__step-description--Color: var(--pf-global--danger-color--100);
```

Comment:
do you still need this var? it isn't used.

Reply:
It's used on line 193 to set `--pf-c-progress-stepper__step-title--Color` for the danger variant.

Reply:
The unused var is `--pf-c-progress-stepper__step--m-danger__step-description--Color`, you're using `--pf-c-progress-stepper__step--m-danger__step-title--Color` on L193 - https://github.com/patternfly/patternfly/blob/5e9e4684fe9c0ae347fb6c30d26509bd1cb82e74/src/patternfly/components/ProgressStepper/progress-stepper.scss#L193

Reply:
Right.